packages/recommend-core/src/getTrendingFacets.ts

```diff
@@ -1,7 +1,7 @@
 import { RecommendClient, TrendingFacetsQuery } from '@algolia/recommend';
 
 import { TrendingFacet } from './types';
-import { mapToRecommendations } from './utils';
+import { mapWithSortToRecommendations } from './utils';
```

Comment:
Here I'm thinking that both methods `mapToRecommendations` and `mapWithSortToRecommendations` are sorting the results. Maybe we could provide a more explicit naming for the latest. For example `mapByScoreToRecommendation`? WDYT?
(This is nitpicking, feel free to keep as is)